debian/kano-updater.postinst

```diff
@@ -17,6 +17,7 @@ case "$1" in
 
         # Create custom sudoers file
         echo "%sudo   ALL=(root) NOPASSWD: /usr/bin/kano-updater" > $TMP_FILE
+        echo "%sudo   ALL=(root) NOPASSWD: /usr/bin/kano-updater-recovery" >> $TMP_FILE
```

Comment:
Can we move away from this `TMP_FILE` mess and instead create a `kano-updater.conf` file in the repo and install it with the `kano-updater.install` file. `dh_fixperms` should already be handling [permissions](https://github.com/Debian/debhelper/blob/master/dh_fixperms#L158) but at the very most, the contents of this `postinst` should set permissions.

Reply:
Also changing this file should trigger a copyright bump.

Reply:
I like the idea of the sudoers conf file. Will tackle that separately

Thanks, fixed the copyright here